.github/workflows/deploy.yml

```
@@ -0,0 +1,190 @@
+      - name: Deploy to server
+        env:
+          IMAGE_URL: ${{ needs.build.outputs.image-url }}
+          GIT_SHA: ${{ needs.build.outputs.git-sha }}
+        run: |
+          ssh -i ~/.ssh/id_rsa ${{ secrets.OCI_USERNAME }}@${{ secrets.OCI_HOST }} << 'ENDSSH'
+            set -e
+
+            echo "📦 Starting deployment..."
+            echo "🏷️  Image: ${IMAGE_URL}"
+            echo "🔖 Version: ${GIT_SHA}"
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)